packages/compass-explain-plan/src/components/explain-tree/explain-tree-stage.tsx

```diff
@@ -120,22 +193,32 @@ const ExecutionStats: React.FunctionComponent<ExecutionstatsProps> = ({
   return (
     <div className={executionStatsStyle}>
       <div>
-        <span>nReturned </span>
-        <span>
-          <Badge>{nReturned}</Badge>
-        </span>
+        <span>Returned </span>
+        <StatsBadge stats={nReturned} />
       </div>
       <div>
         <span>Execution Time</span>
         <span>
-          <Clock
-            prevStageExecTimeMS={prevStageExecTimeMS}
-            curStageExecTimeMS={curStageExecTimeMS}
-            totalExecTimeMS={totalExecTimeMS}
-            width={60}
-            height={60}
-            className={clockStyles}
-          />
+          <Tooltip
+            align="top"
+            justify="middle"
+            trigger={({ children, ...props }) => (
+              <div {...props} className={clockStyles}>
+                {children}
+                <Clock
+                  prevStageExecTimeMS={prevStageExecTimeMS}
+                  curStageExecTimeMS={curStageExecTimeMS}
+                  totalExecTimeMS={totalExecTimeMS}
+                  width={60}
+                  height={60}
```

Comment:
This seems to be 50 in the designs

![image](https://github.com/mongodb-js/compass/assets/5036933/940c2e22-8963-4256-80a3-c7aa79b4744f)